task3.js

```diff
@@ -0,0 +1,12 @@
+const csvtojson = require('csvtojson');
+let fs = require('fs');
+
+csvtojson()
+  .fromFile('./csv/file.csv')
+  .then((json) => {
```

Comment:
As per AC3:
Do not load all the content of the csvfile into RAM via stream (read/write file content line by line). In case of read/write errors, log them in the console. The program should be started via npm script using nodemon(i.e. npm run task2).
For really huge csv file (for me it's longer than 6k lines), your app doesn't parse json correctly

Reply:
Additionally (optional), please, consider making output format a real JSON, not a bunch on JSON-like line

Reply:
Done.